test/run-adapter-specific-tests.js

```diff
@@ -203,7 +203,7 @@ describe('dontUseObjectIds', function() {
 
       it('should find the records correctly', function(done) {
 
-        models.user._adapter.datastores.test.manager.collection('user').insert([{_id: 123, name: 'sid'}, {_id: 555, name: 'nancy'}], function(err) {
+        models.user._adapter.datastores.test.manager._WPdb.collection('user').insertMany([{_id: 123, name: 'sid'}, {_id: 555, name: 'nancy'}], function(err) {
```

Comment:
Is this how end users would do native stuff in mongo?  If so, would it be better to give them a clearer path-- i.e. some way that doesn't involve accessing `_WPdb`?  That way they know it's ok that theyre doing it